bench/browser-bench.js

```diff
@@ -0,0 +1,36 @@
+import bench from "do-you-even-bench";
+import Benchmark from "benchmark";
+window.Benchmark = Benchmark;
+
+import DebounceCancel from "./benches/debounce-cancel";
+import LaterCancel from "./benches/later-cancel";
+import ScheduleCancel from "./benches/schedule-cancel";
+import ScheduleFlush from "./benches/schedule-flush";
+import ThrottleCancel from "./benches/throttle-cancel";
+
+var suites = [];
+[
+  DebounceCancel,
+  LaterCancel,
+  ScheduleCancel,
+  ScheduleFlush,
+  ThrottleCancel,
+].forEach(function (module) {
+  if (Array.isArray(module)) {
+    suites = suites.concat(module);
+  } else {
+    suites.push(module);
+  }
+});
```

Comment:
Could this be reduced to:

```suggestion
const suites = [
  DebounceCancel,
  LaterCancel,
  ScheduleCancel,
  ScheduleFlush,
  ThrottleCancel,
].flat();
```

?

Reply:
Yup that's much cleaner - thanks!